src/clang_parser.cpp

```diff
@@ -104,8 +104,19 @@ static SizedType get_sized_type(CXType clang_type)
       {
         return SizedType(Type::string, size);
       }
-      // TODO add support for arrays
-      return SizedType(Type::none, 0);
+
+      // Prevent unbounded recursion from get_sized_type call
```

Comment:
Does this stop us indexing into multi-dimensional arrays? When might we get unbounded recursion here?

Reply:
I'll try out a multi-dimensional array for a test case.

I'll admit i was being a bit timid, I just wanted to be certain there was a base case for my recursion.

Reply:
@ajor I've decided for expediency to just support one-dimensional arrays for now. It wasn't trivial (so far as I could see) to support multi-dimensional arrays at the moment (I didn't investigate the coredump very much), but I think it should be out of scope of this PR for now.

I've updated the comment to indicate that 1-D arrays are only supported for now.